CMakeLists.txt

```diff
@@ -412,7 +417,7 @@ elseif(ANDROID)
 	set(_NBL_PLATFORM_ANDROID_ 1)
 elseif(UNIX)
 	set(_NBL_PLATFORM_LINUX_ 1)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.1")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.1 -fpermissive")
```

Comment:
@Nihon13 what did you need to enable permissive for?

Reply:
Without -fpermissive flag, i couldn't build `CDirQuantCacheBase.h` file.
GCC error: there are no arguments to ‘getSerializedCacheSizeInBytes’ that depend on a template parameter, so a declaration of ‘getSerializedCacheSizeInBytes’ must be available [-fpermissive]

Reply:
that flag needs to get removed and that compile error fixed, maybe @Crisspl can help here.

Reply:
so just move definition/forward decl of getSerializedCacheSizeInBytes over its usages. If this doesnt help, id have to see whole exact build log

Reply:
Fixed.